cmd/snap-exec/main_test.go

```diff
@@ -215,7 +215,7 @@ func (s *snapExecSuite) TestSnapExecAppRealIntegration(c *C) {
 	})
 
 	canaryFile := filepath.Join(c.MkDir(), "canary.txt")
-	script := filepath.Join(dirs.GlobalRootDir, "/snap/snapname/42/run-app")
+	script := fmt.Sprintf("%s/snapname/42/run-app", dirs.SnapMountDir)
```

Comment:
It was just looking for the easiest way to do it. I guess I could switch to `filepath.Join` if there's a desire to do that.